src/components/Card.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import styled from 'styled-components'
+
+function Card({ text, author }) {
+  return (
+    <>
```

Comment:
I don't think that you need a fragment, because you have `<StyledCards>` which contains all other components and contents.